### ISSUE TITLE
Feature request: Blacklist.

### ISSUE THREAD

**Issue by RoosDaniel** [j-delaney/back-to-backspace#32]
Hello!

Love the extension. Just one request: there should be a blacklist for pages that you don't want for the extension to be activated on.

Thanks!

**Comment by tobibeer** [comment]
Yes, absolutely, I am going mad over going back in TFS when actually in any editor... or, something about TFS is different where this useful extension should not kick-in when in any input field.
